maker-frontend/src/MakerApp.tsx

```diff
@@ -105,20 +115,39 @@ export default function App() {
                             value={format(maxQuantity)}
                         />
 
-                        <Text>Offer Price:</Text>
+                        <Text>Maker Short Price:</Text>
```

Comment:
I'm still super confused about the naming... 
Maker short price means, that's the price the maker offers to a taker who goes long?

Reply:
Yes. That's why I tried do be more elaborate - I could go even more verbose "Taker going long price" instead if that helps.